packages/cli/src/constants.ts

```diff
@@ -56,29 +57,43 @@ export const CollaboratorEmailNode = new QTreeNode({
 });
 
 export const ManifestFilePathParamName = "manifest-file-path";
-export const OutputZipPathParamName = "output-zip-path";
-export const OutputManifestParamName = "output-manifest-path";
+export const AppPackageFilePathParamName = "app-package-file-path";
 export const BuildPackageOptions: OptionsMap = {
   [ManifestFilePathParamName]: {
     type: "string",
     global: false,
     description:
       "Select the Teams app manifest template path, default to '${folder}/appPackage/manifest.json'",
   },
-  [OutputZipPathParamName]: {
+  [CoreQuestionNames.OutputZipPathParamName]: {
     type: "string",
     global: false,
     description:
-      "Select the output path of the zipped app package, default to '${folder}/build/appPackage/appPackage.${env}.json'",
+      "Select the output path of the zipped app package, default to '${folder}/build/appPackage/appPackage.${env}.zip'",
   },
-  [OutputManifestParamName]: {
+  [CoreQuestionNames.OutputManifestParamName]: {
     type: "string",
     global: false,
     description:
       "Select the output path of the generated manifest path, default to '${folder}/build/appPackage/manifest.${env}.json'",
   },
 };
 
+export const ValidateApplicationOptions: OptionsMap = {
+  [ManifestFilePathParamName]: {
+    type: "string",
+    global: false,
+    description:
+      "Select the Teams app manifest template path, default to '${folder}/appPackage/manifest.json'. Validate using manifest schema.",
```

Comment:
Select the input Teams app manifest file path, defaults to '${folder}/appPackage/manifest.json'. It will be validated using manifest schema.",